verify/verify-pylint.sh

```diff
@@ -17,6 +17,9 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+export PYLINTHOME=$(mktemp -d)
+export PATH="${TEST_SRCDIR}/pylint:${PATH}"
```

Comment:
maybe handle `TEST_SRCDIR` not being set?

my original suggestion was intended so that `verify-pylint.sh` would still work outside bazel.